package.json

```diff
@@ -1,26 +1,49 @@
 {
   "name": "saa_victoria_subtheme",
-  "version": "1.0.0",
+  "version": "8.1.6",
   "repository": "https://github.com/SU-SWS/saa_victoria_subtheme",
   "scripts": {
     "build": "webpack --mode=development --display-error-details",
     "publish": "webpack --mode=production",
     "watch": "webpack --mode=development --watch",
     "lint:js": "eslint -c .eslintrc 'src/js'",
-    "lint:sass": "sass-lint -c .sass-lint.yml '**/*.scss' -v -q -i 'vendor/**/*.scss, node_modules/**/*.scss'"
+    "lint:sass": "sass-lint -c .sass-lint.yml '**/*.scss' -v -q -i 'vendor/**/*.scss, node_modules/**/*.scss'",
+    "release:bump": "npm version prerelease --preid=dev",
+    "release:bump-patch": "npm version patch",
+    "release:bump-minor": "npm version minor",
+    "release:bump-major": "npm version major",
+    "release:changelog": "auto-changelog -p --tag-prefix='' && git commit -am \"Updated Changelog\"",
+    "release:yaml": "node ./dev/bump-yaml.js && git commit -am \"Updated Info.yml\"",
+    "release": "release-it --ci",
+    "release:dev": "release-it --ci --github.preRelease --preReleaseId=dev --npm.tag=dev",
+    "release:manual": "release-it"
+  },
+  "release-it": {
+    "git": {
+      "changelog": "auto-changelog --unreleased-only --stdout",
+      "requireUpstream": false
+    },
+    "github": {
+      "release": true,
+      "releaseNotes": "auto-changelog --unreleased-only --stdout"
+    },
+    "npm": {
+      "publish": false
+    }
   },
   "devDependencies": {
-    "@babel/polyfill": "^7.8.3",
-    "@babel/core": "^7.8.6",
-    "@babel/preset-env": "^7.8.6",
-    "autoprefixer": "^9.7.4",
-    "babel-loader": "^8.0",
+    "@babel/core": "^7.12.13",
```

Comment:
A number of these npm packages had vulnerabilities, not that it matters as these don't actually run in production, but I felt it was good to update them. `npm update`